duckduckgo/iOS
# BSK update for JS pixels

Task/Issue URL: https://app.asana.com/0/0/1203478208847847/f
Tech Design URL:
CC:

**Description**:

This PR adds support for a new BSK delegate method that allows the JS layer to send autofill pixels through.



**Steps to test this PR**:
1. The JS pixel sent by Autofill is only used for identities, so does not yet apply to iOS directly. To test this, run through Autofill login and email feature testing, and verify that no extra pixels are fired.



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
